numpy/core/__init__.py

```diff
@@ -52,6 +52,16 @@
 del env_added
 del os
 
+# Test that multiarray is a pure python module wrapping _multiarray_umath,
+# and not the old c-extension module.
+if not getattr(multiarray, '_multiarray_umath', None):
+    raise ImportError('''Something is wrong with the NumPy installation.
+While importing 'multiarray' we detected an older version of numpy. One
```

Comment:
This would look better broken into a series of strings.
```
    raise ImportError("Something is wrong with the NumPy installation. While "
                      "importing 'multiarray' ... "
                      "etc.")
```

Reply:
reformatted and force-pushed